modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -154,6 +154,10 @@ echo "Forseti Startup - Enabling and starting Forseti service."
 systemctl enable --now forseti
 echo "Forseti Startup - Success! The Forseti API server has been enabled and started."
 
+# Increase Open File Limit
+echo "ubuntu soft nofile 32768" | sudo tee -a /etc/security/limits.conf
+echo "ubuntu hard nofile 32768" | sudo tee -a /etc/security/limits.conf
```

Comment:
If someone resets the VM, will this be added to the `limits.conf` file again?

Reply:
Have verified this on `red2k18-terraform-test project`. 
1. Upon resetting the server VM, the new ulimit is applied. 
2. Upon manually removing the code in this PR from the server VM and resetting it, it stays and doesn't reset to 1024 as there is no configuration that overrides 32768. 
3. To change the ulimit to a different value, user needs to edit the ulimit configuration in the startup script of the server VM and reset the VM.

Reply:
Just wanted to add that I have tested on `rehana-test` project as well. New ulimit code is picked up after VM reset. Also, inventory is successfully created after modifying the startup script.